composer.json

```
@@ -39,12 +39,7 @@
         "psr/http-message": "^1.0 || ^2.0",
         "psr/http-message-implementation": "^1.0",
         "psr/http-server-middleware": "^1.0",
-        "web-token/jwt-checker": "^2.0 || ^3.0",
-        "web-token/jwt-core": "^2.0 || ^3.0",
-        "web-token/jwt-encryption": "^2.0 || ^3.0",
-        "web-token/jwt-key-mgmt": "^2.0.7 || ^3.0",
-        "web-token/jwt-signature": "^2.0 || ^3.0",
-        "web-token/jwt-signature-algorithm-rsa": "^2.0 || ^3.0"
+        "web-token/jwt-library": "^2.0 || ^3.0"
```

This doesn't make sense, this package exists only since 3.3: https://packagist.org/packages/web-token/jwt-library
```suggestion
        "web-token/jwt-library": "^3.3"
```